ChangeLog.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## Version XX.XX.XX
+
+### New features
+
+1. SAS tokens are supported on HNS (Hierarchical NameSpace/Azure Data Lakes Generation 2) Storage Accounts

```

Comment:
Namespace is normally written with lower cases. 

Lake is singular in Data Lake.  (Better fix, to get product names correct ;-)